openmrs-19/src/main/java/org/motechproject/openmrs19/util/ConverterUtils.java

```diff
@@ -428,6 +426,7 @@ public static Patient toPatient(OpenMRSPatient patient, OpenMRSPerson savedPerso
         Patient converted = new Patient();
         Person person = new Person();
         person.setUuid(savedPerson.getPersonId());
+        person.getPreferredAddress().setPreferred("true");
```

Comment:
I'm not sure about preferred field. In OpenMRS server Person has fields addresses and preferredAddress. PreferredAddress is one of object in addresses list with flag preferred set to true. In our Motech implementation we support only preferredaddress. 

1. I noticed that you set this flag in method toPatient. Here you only set person id in patient object. You should do this in toPerson method. 
2. It should be boolean field not string
3. Please check if this field is required. I suppose that OpenMRS module stores address as preferred and this flag is set automatically